avm/res/automation/automation-account/main.bicep

```diff
@@ -562,3 +611,156 @@ type gallerySolutionType = {
   @description('Required. Plan for solution object supported by the OperationsManagement resource provider.')
   plan: solutionPlanType
 }
+
+type pwsh72ModuleType = {
+  @description('Required. Name of the Powershell72 Automation Account module.')
+  name: string
+
+  @description('Optional. The location to deploy the module to.')
+  location: string?
+
+  @description('Optional. Tags of the Powershell 72 module resource.')
+  tags: object?
+
+  @description('Required. Module package URI, e.g. https://www.powershellgallery.com/api/v2/package.')
+  uri: string
+
+  @description('Optional. Module version or specify latest to get the latest version.')
+  version: 'latest'?
+}[]?
```

Comment:
For this and the other new types, please move the `[]?` to the parameter and remove it from the type. 
The reason being: It makes the parameters easier to understand as you can immediately see that a parameter is, for example, an array, without needing to look their type up first.

Reply:
Done.